metricbeat/mb/lightmetricset.go

```diff
@@ -96,6 +100,16 @@ func (m *LightMetricSet) Registration(r *Register) (MetricSetRegistration, error
 	return registration, nil
 }
 
+func (m *LightMetricSet) useHostURISchemeIfPossible(host, uri string) string {
```

Comment:
This worths a comment :slightly_smiling_face:

Reply:
Comment added.

Reply:
Thanks!